Expecto/Expecto.fs

```diff
@@ -1129,9 +1132,15 @@ module Impl =
     let asMembers x = Seq.map (fun m -> m :> MemberInfo) x
     let bindingFlags = BindingFlags.Public ||| BindingFlags.Static
     fun (t: Type) ->
+#if RESHAPED_REFLECTION
```

Comment:
Can't this be used by default, every time? Don't these APIs exist on Mono 4.6.2/.Net 4.6.1 and upwards?